rover_antenna/src/arbitration.cpp

```diff
@@ -0,0 +1,104 @@
+#include "rclcpp/rclcpp.hpp"
+#include "rover_msgs/msg/antenna_cmd.hpp"
+#include "rover_msgs/srv/antenna_arbitration.hpp"
+
+using namespace std::chrono_literals;
```

Comment:
Utilise le nom complet ou tu utilise ça:
Example:
timer_ = this->create_wall_timer(100ms, std::bind(&Arbitration::timer_callback, this));
devient
timer_ = this->create_wall_timer(std::chrono::milliseconds(100), std::bind(&Arbitration::timer_callback, this));

Reply:
fait